### DISCUSSION TITLE
Arbitrum NOVA (Reddit's Fav Chain)

### DISCUSSION THREAD

**Discussion by communityvolunteer** [wevm/wagmi#2362] [Ideas]
Arbitrum Nova (Reddit's fav chain) Hey all, could you guys add support for Arbitrum Nova, the chain reddit uses? Thanks :)

**Comment by afa7789** [comment]
Any update on this ?

**Reply by tmm** [reply]
Anyone is welcome to add: https://github.com/wagmi-dev/references

**Reply by afa7789** [reply]
@tmm https://github.com/wagmi-dev/references/blob/main/packages/chains/src/arbitrumNova.ts this is added already, how can I use the wagmi version which supports it ?

**Reply by tmm** [reply]
Needs to be exported from [core](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/src/chains.ts) and [react](https://github.com/wagmi-dev/wagmi/blob/main/packages/react/src/chains.ts) entrypoints.

**Reply by pedromcunha** [reply]
Created a PR here for it: https://github.com/wagmi-dev/wagmi/pull/2492
